apps/nextjs/src/components/reviews.tsx

```diff
@@ -83,10 +88,29 @@ export function CreateReviewForm() {
   });
 
   const onSubmit = (data: z.input<typeof ReviewFormSchema>) => {
-    createReview.mutate(CreateReviewSchema.parse(data));
+    createReview.mutate(
+      CreateReviewSchema.parse({
+        ...data,
+        restaurantId: autocompleteData,
+        name: restoname,
+      }),
+    );
   };
 
-  return <ReviewForm form={form} onSubmit={onSubmit} />;
+  return (
+    <div>
+      <div>
+        <SearchBox
+          onSelectAddress={(address, place_id) => {
```

Comment:
```suggestion
          onSelectAddress={(address, placeId) => {
```
try and use camelCase over snake_case